examples/integrations/cdi/datasource-hikaricp-mysql/src/main/docker/Dockerfile

```diff
@@ -0,0 +1,27 @@
+#
+# Copyright (c) 2018 Oracle and/or its affiliates. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+FROM openjdk:8-jre-alpine
```

Comment:
FYI, I plan on changing the quickstart examples to use `openjdk:8-jre-slim` since rumor is JDK 8 works on Alpine only by accident.